viewing_party/main.py

```diff
@@ -0,0 +1,161 @@
+def create_movie(movie_title, genre, rating):
+
+    # Create a new movie
+    new_movie = {"title": movie_title, "genre": genre, "rating": rating}
+
+    if new_movie["title"] == None or new_movie["genre"] == None or new_movie["rating"] == None:
+        return None
+    return new_movie
+
+def add_to_watched(user_data, movie):
+
+    # Add movie in the list of movies that the user has already watched
+    user_data = {
+        "watched": [movie]
+    }
+    return user_data
```

Comment:
this works, but it suggests that `user_data["watched"]` will only ever have one item in its list, ie `[movie]` and it will replace any others that were previously there

```suggestion
    user_data["watched"].append(movie)
    return user_data
```